frontend/src/metabase/containers/SaveQuestionModal.unit.spec.tsx

```diff
@@ -80,6 +84,13 @@ const setup = async (
 
   const settings = mockSettings({ "enable-query-caching": isCachingEnabled });
 
+  const state = createMockState({
```

Comment:
`SaveQuestionModal` now relies on a new selector, so I need to mock the state to reflect the change.

https://github.com/metabase/metabase/blob/4217c360cfe5770c1cc28e766bf6d926a4aabbe6/frontend/src/metabase/query_builder/selectors.js#L559-L569